src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/controller/ReferralController.kt

```diff
@@ -138,9 +138,12 @@ class ReferralController(
     val userID = authentication.token.getClaimAsString("user_id")
       ?: throw ServerWebInputException("no 'user_id' claim in authentication token")
 
+    val userName = authentication.token.getClaimAsString("user_name")
+      ?: throw ServerWebInputException("no 'user_name' claim in authentication token")
+
     val authSource = authentication.token.getClaimAsString("auth_source")
       ?: throw ServerWebInputException("no 'auth_source' claim in authentication token")
 
-    return AuthUser(id = userID, authSource = authSource)
+    return AuthUser(id = userID, authSource = authSource, userName = userName)
```

Comment:
_nitpick_
`userName` and `username` are used throughout the PR -- should we pick one style for consistency?

Reply:
this is a good spot. george raised something similar on a backend PR. i'll update this.